### ISSUE TITLE
Emscripten: Broken custom menu in The Sacred Tears: TRUE

### ISSUE THREAD

**Issue by ghost** [EasyRPG/Player#2294]
The key input processing fix based on engine version fails with The Sacred Tears: TRUE

The pull request #2143 contains the breaking fix designed to amend #2140.

Last working version: https://easyrpg.org/play/pr2137/?game=tstt
First failing version: https://easyrpg.org/play/pr2143/?game=tstt

Shift key and arrows don't work on the custom menu, only decision key and cancel key work.

**Note:** this has been tested on Emscripten only, RPG_RT.exe detection might differ.

Engine detection details:
```
Debug: Using RPG2k Interpreter
Debug: Could not get the size of RPG_RT.exe
Debug: Assuming older engine
Debug: RPG2k < v1.50 / RPG2k3 < v1.05 detected
Debug: Engine configured as: 2k=1 2k3=0 MajorUpdated=0 Eng=0
```

The detection seems wrong because this game uses shift key and 2k < 1.50 didn't support it.
A possible better heuristic should detect chunk size differences (if possible) before running key input processing.

Despite of mixed versioned games such .flow and other cases for menus, chunk size here should be reliable when checking RPG_RT.exe is not possible, or improve the Emscripten port to properly fetch the RPG_RT.exe size.

**Comment by fdelapena** [comment]
The game mentioned gets detected properly on desktop. The MP3 detection was the key. This does not seem performed from Emscripten (it could check it from the index.json nowadays).

**Comment by Ghabry** [comment]
yeah I already thought about reading from index.json when Exists() is called. Also useful for 2k3e engine detection. But I wanted to do this using a EmscriptenVirtualFilesystem implementation to keep the code clean so this is after my FileFinder refactor. Sorry :|

**Comment by Ghabry** [comment]
Gray Garden is also affected:
Emscripten tests:
https://easyrpg.org/play/?game=the_gray_garden
https://easyrpg.org/play/master/?game=the_gray_garden

**Comment by fdelapena** [comment]
Got a recent mail bug report from a game in Korean with another possible similar bug, but this time not from emscripten.

**Comment by Ghabry** [comment]
Uhm yeah guess with nicovideo hosting us now the autodetection should be improved a bit :sweat_smile: 

Will likely use some kind of VFS layer for this, makes faking of certain behaviours (e.g. Existance check through index.json) easier.
